### PR TITLE
Handle, migrate data is available initially in product to ES, change …

### PR DESCRIPTION
**Pull request by DieuNguyenThi**
Hi all,

Plz help me to review this PR

Thanks!

### CODE REVIEW

File: connectors-config/connections/source.json [hunk 1]
```diff
@@ -1,14 +1,13 @@
 {
   "connector.class": "io.debezium.connector.postgresql.PostgresConnector",
-  "topic.prefix": "dbserver1",
+  "topic.prefix": "DBProduct",
```
Comment (thiennn) [inline connectors-config/connections/source.json:3]: dbproduct seems better

File: connectors-config/connections/source.json [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
   "database.user": "admin",
   "database.dbname": "product",
```
Comment (thiennn) [inline connectors-config/connections/source.json:5]: can we filter the tables, I see at the moment all the kafka connect create 1 topic for every table

**Review by LEHOANGGLAM — APPROVED** [review]

File: docker-compose.yml [hunk 3]
```diff
@@ -143,7 +143,7 @@ services:
       SERVER_PORT: "80"
   search:
     image: ghcr.io/nashtech-garage/yas-search:latest
-    entrypoint: ["./wait-for-it.sh", "elasticsearch:9200", "--timeout=180", "--", "java", "-jar", "/search-0.0.1-SNAPSHOT.jar"]
+    entrypoint: ["./wait-for-it.sh", "elasticsearch:9200", "kafka:9092", "--timeout=180", "--", "java", "-jar", "/search-0.0.1-SNAPSHOT.jar"]
```
Comment (thiennn) [inline docker-compose.yml:146]: We should increase the timeout to 300 just in case the computer is slow

**Comment by thiennn** [comment]
Btw, we should update guide to run  ./start-source-connectors.sh in the README

File: connectors-config/connections/source.json (hunk 2 above)
Comment (DieuNguyenThi) [reply connectors-config/connections/source.json:5]: I think we can exclude tables which don't need to be tracked, I've tried to exclude but still no success. I will find out more about this issue.

**Review by nguyenvanhadncntt — APPROVED** [review]

**Pull request merged**